src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -278,13 +278,26 @@ public void writeBody(PrintWriter w) {
                             }
                             if (!branchFound) continue;
                             urlFound = true;
-
                             if (!(project instanceof AbstractProject && ((AbstractProject) project).isDisabled())) {
+                                //JENKINS-30178 Add default parameters defined in the job
+                                if (project instanceof Job) {
+                                        List<ParameterValue> jobParametersValues = getDefaultParametersValues((Job) project);
+                                        for (ParameterValue parameterValue : urlBuildParameters) {
+                                            for (ParameterValue defaultParameterValue : jobParametersValues) {
+                                                if (!parameterValueExist(urlBuildParameters, defaultParameterValue.getName())) {
+                                                    allBuildParameters.add(defaultParameterValue);
+                                                }
+                                            }
+                                        }
+                                }
+
+                                urlBuildParameters = allBuildParameters;
```

Comment:
:ant: not required according to the code below. just use <code> allBuildParameters</code>